products/centreon.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Centreon
+category: server-app
+tags: php-runtime
+permalink: /centreon
+versionCommand: cat /etc/centreon/centreon.conf
+releasePolicyLink: https://docs.centreon.com/docs/releases/lifecycle/
+releaseDateColumn: true
+activeSupportColumn: true
+eolColumn: Security Support
+
+#identifiers:
+#-   repology: zabbix
+#-   purl: pkg:brew/zabbix
+#-   purl: pkg:github/zabbix/zabbix
+
+auto:
+-   git: https://github.com/centreon/centreon.git
+
+# For non-LTS releases :
+# - support(x) = release(x) + 6 months
+# - eol(x) = release(x) + 7 months
+releases:
+-   releaseCycle: "23"
+    releaseDate: 2023-04-01
+    support: true
+    eol: 2025-04-01
+    latest: "23.04"
+    latestReleaseDate: 2023-04-01 
+    link: https://www.centreon.com/centreon-23-04-discover-the-latest-release/
+
+-   releaseCycle: "22"
+    releaseDate: 2022-05-01
+    support: true
+    eol: 2024-10
+    latest: "22.10"
+    latestReleaseDate: 2022-10
+    link: https://www.centreon.com/centreon-fall22-whats-new-in-the-22-10-software-version/
+
+-   releaseCycle: "21"
+    releaseDate: 2021-04-01
+    support: true
+    eol: 2023-11
+    latest: "21.10"
+    latestReleaseDate: 2021-11
+    link: https://docs.centreon.com/docs/21.10/releases/introduction/
+
+-   releaseCycle: "20"
+    releaseDate: 2020-04-01
+    support: false
+    eol: 2022-05
+    latest: "20.10"
+    latestReleaseDate: 2020-10
+    link: https://www.centreon.com/new-software-release-centreon-20-10/
+
+-   releaseCycle: "19"
+    releaseDate: 2019-04
+    support: falase
```

Comment:
```suggestion
    support: false
```

Reply:
And please have a look at the deploy logs on https://app.netlify.com/sites/endoflife-date/deploys/650b7e989f327a000777d16e: there are a lot of errors in that page.